components/miro_custom_app/actions/get-items/get-items.mjs

```diff
@@ -0,0 +1,33 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  name: "Get Items",
+  version: "0.0.1",
+  key: "get-items",
+  description: "Returns items on a Miro board",
+  type: "action",
+  props: {
+    miro_custom_app: {
+      type: "app",
+      app: "miro_custom_app"
+    },
+    boardId: {
+      type: "string",
+      description: "Board ID",
+      optional: false,
+    }
```

Comment:
Here you can create an `async options()` prop to get a dropdown menu to select a `boardId` using the api.